API/Model/CountryRepository.cs

```diff
@@ -36,9 +37,19 @@ public IEnumerable<Country> GetAllCountries()
 
         public Country? GetCountryOfTheDay()
         {
-            // TODO: Implement logic to get country of the day
+            
+            DateTime today = DateTime.Today;
+            string stringToBeHashed = today.Month.ToString() + today.Day.ToString();
+            int hashed = stringToBeHashed.GetHashCode();
+
+
+            List<Country> Countries = _dbContext.Countries.ToList();
```

Comment:
Private variables should start with a lower case letter and all of these could use `var`

Reply:
This is still not fixed and is not consistent with either the style guide or the rest of the project @Huffla